charts/codesealer/README.md

```diff
@@ -13,33 +13,44 @@ In the following we assume that the access token is set in the following way:
 export CODESEALER_TOKEN=<access token>
 ```
 
+This installation requires a Kubernetes Cluster with kubectl.  
+
+### Ingress
+
+To use this Helm chart you will also need to set the following variables to match
+your Ingress Controller's deployment on your Kubernetes Cluster:
+
+```bash
+export INGRESS_NAMESPACE=ingress-nginx
+export INGRESS_DEPLOYMENT=ingress-nginx-controller
+export INGRESS_PORT=443
+export INGRESS_HELM_REPO=https://kubernetes.github.io/ingress-nginx
+export INGRESS_HELM_CHART=ingress-nginx
+```
+
 Additionally you will need an ingress and an application to protect. Below are steps to
 get started with a demo application and an Nginx Ingress. For guides on how to use this
 Helm chart with specific Kubernetes implementations, see the ["Kubernetes Implementation
 Specifics"](#kubernetes-implementation-specifics) section.
 
-### Ingress
-
```

Comment:
I can't get these instructions to work with minikube. I'm assuming the right way will be to use NodePort as with Kind. This, however, seems to have the side effect of disabling access to kubectl in some cases.

Reply:
Minikube still a work in progress.  Will go there next.